aws_crt/iot.py

```diff
@@ -85,13 +85,13 @@ def __init__(self, clientID, useWebsocket=False, cleanSession=True):
 
         self._username = None
         self._password = None
+        self._clientID = clientID
 
         self._elg = io.EventLoopGroup(1)
         self._bootstrap = io.ClientBootstrap(self._elg)
```

Comment:
kinda unrelated, but PyLint on my machine is complaining about "io.EventLoopGroup" since neither "io" or "EventLoopGroup" is explicitly imported